README.md

```diff
@@ -112,15 +112,74 @@ A detailed list of options can be found [here](https://razorpay.com/docs/payment
 _razorpay.open(options);
 ```
 
+## API
+
+### Razorpay
+
+#### open(map<String, dynamic> options)
+
+Open Razorpay Checkout. 
+
+The `options` map has `key` as a required property. All other properties are optional. 
+For a complete list of options, please click [here](https://razorpay.com/docs/payment-gateway/integrations-guide/checkout/standard/#checkout-form).
+
+#### on(String eventName, Function listener)
+
+Register event listeners for payment events.
+
+- `eventName`: The name of the event.
+- `listener`: The function to be called. The listener should accept a single argument of the following type:
+  - [`PaymentSuccessResponse`](#paymentsuccessresponse) for `EVENT_PAYMENT_SUCCESS`
```

Comment:
Are these available as static properties of some class (like `Razorpay.EVENT_PAYMENT_SUCCESS`?

Reply:
Saw that this is available.